FLT/for_mathlib/IsCentralSimple.lean

```diff
@@ -38,7 +56,51 @@ theorem MatrixRing.isCentralSimple (ι : Type v) (hι : Fintype ι) (hnonempty :
     convert mem_range_scalar_of_commute_stdBasisMatrix (M := d) fun i j hij => hd _
     simp_rw [Set.mem_range, eq_comm, algebraMap_eq_diagonal, Pi.algebraMap_def,
       Algebra.id.map_eq_self, scalar_apply]
-  is_simple := sorry
+  is_simple.exists_pair_ne := by
+    use ⊥, ⊤
+    apply_fun (· 0 1)
+    convert false_ne_true
+    exact iff_false_iff.mpr zero_ne_one
```

Comment:
Please make a note to clean this up after https://github.com/leanprover-community/mathlib4/pull/12860 lands